inductiva/tests/resources/test_machines.py

```diff
@@ -0,0 +1,52 @@
+"""Unit tests for the Computational Resources classes."""
+from unittest import mock
+import inductiva
+
+
+def fake_register(self, **kwargs):  # pylint: disable = unused-argument
+    self._id = "id-resource"  # pylint: disable = protected-access
+    self._name = "name-resource"  # pylint: disable = protected-access
+    self.register = False
+
+
+@mock.patch.object(inductiva.resources.MPICluster,
+                   attribute="_register_machine_group",
+                   new=fake_register)
+def test_machines__mpicluster__register():
+    """Check the registering of a MPICluster.
+    
+    Goal: Verify that the MPICluster is initializating and the registration is
+    processed, by mocking it instead of calling the API.
+    """
+    inductiva.api_key = "dummy"
```

Comment:
why not set this at the top of the the module?

```python
"""Unit tests for the Computational Resources classes."""
from unittest import mock
import inductiva

inductiva.api_key = "dummy"

...
```

BTW, why not have a key for these tests? It can be a secreted inject by github when running the tests.

Reply:
This key isn't used for API calls so no need to have it being a secret.